### PR TITLE
Add further clarification about ANSI to docs

### PR DESCRIPTION
**Pull request by colebow**
## Description

People don't really know what it means to abide by the ANSI spec, so let's call out that we're compliant, that we haven't fully implemented, and make it clearer why that matters.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

### CODE REVIEW

**Review by mosabua — CHANGES_REQUESTED** [review]
I think we might have to clarify that ANSI SQL comes in editions over the years, and that we also strive to follow other well adopted conventions when ANSI SQL has gaps. That specifically also applies to functions (think geospatial and PostGIS, and many others). Also maybe this has to be more than one paragraph.

File: docs/src/main/sphinx/language.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # SQL language
 
-Trino is an ANSI SQL compliant query engine. This standard compliance allows
-Trino users to integrate their favorite data tools, including BI and ETL tools
-with any underlying data source.
+Trino is an ANSI SQL compliant query engine. ANSI compliance means that all SQL
+language implemented in Trino adheres to the ANSI SQL specification, but not
```
Comment (ebyhr) [inline docs/src/main/sphinx/language.md:4]: > all SQL language implemented in Trino adheres to the ANSI SQL specification

This is false. For instance, `COMMENT` statement doesn't exist in ANSI.

**Comment by mosabua** [comment]
@colebow I assume you will continue on this ;-)

**Comment by mosabua** [comment]
👋 @colebow ... are you taking this up again some time ?

**Comment by colebow** [comment]
I'll push an update to this sometime next week and we can get it merged yeah

**Comment by mosabua** [comment]
Sure .. just let me know when I should look ;-)

**Pull request closed without merging**